tests/datasets/configuration/entity_with_ip_column_mappers.yaml

```diff
@@ -0,0 +1,71 @@
+version: v1
+kind: entity
+name: generic_metrics_sets
+
+schema:
+  [
+    { name: org_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: metric_id, type: UInt, args: { size: 64 } },
+    { name: timestamp, type: DateTime },
+    { name: bucketed_time, type: DateTime },
+  ]
+
+readable_storage: generic_metrics_sets
+writable_storage: generic_metrics_sets_raw
+query_processors:
+  -
+    processor: TagsTypeTransformer
+  -
+    processor: ReferrerRateLimiterProcessor
+translation_mappers:
+  columns:
+    -
+      mapper: ColumnToIPAddress
+      args:
+        from_table_name:
```

Comment:
Is it necessary to have these empty values? Could we maybe set a reasonable default so they don't need to be defined in the YAML?

Reply:
Right now, these empty values are also required by other mappers (subscriptables, functions, curried_functions). I agree, we should set some reasonable default either as a config override or within the class constructor itself. Since this PR is blocking some other work, I will create a ticket for this issue.